TwoNote/app/src/main/java/com/microsoft/device/display/samples/twonote/model/INode.kt

```diff
@@ -18,6 +18,12 @@ class INode(
     var id: Int = 0
 ) : Serializable {
 
+    val dateModifiedString: String
+        get() {
+            return dateModified.format(DateTimeFormatter.ofLocalizedDateTime(FormatStyle.SHORT))
+        }
```

Comment:
is there any advantage to making this a val instead of just having a getter method that returns the date as a string?

Reply:
I don't think there's any difference (besides having to use parentheses haha) so i can change it to a getter if that's more efficient for the filesaving stuff!